build.sbt

```diff
@@ -150,8 +151,17 @@ onLoad.in(Global) ~= { old =>
 cancelable.in(Global) := true
 crossScalaVersions := Nil
 
+lazy val execScalafmt = taskKey[Unit]("Execute scalafmt executable in /bin")
+execScalafmt := {
+  "bin/scalafmt" !
+}
+
 addCommandAlias("scalafixAll", "all compile:scalafix test:scalafix")
 addCommandAlias("scalafixCheck", "; scalafix --check ; test:scalafix --check")
+addCommandAlias(
+  "preparePr",
+  "all compile:scalafix test:scalafix ; execScalafmt"
```

Comment:
can you use alias here? `all compile:scalafix test:scalafix` is defined as `scalafixAll`

Reply:
Ahh, jeez, it was right above and didn't use it 😆 

Yea, I can switch this.